torch/quantization/fx/quantize.py

```diff
@@ -463,7 +465,7 @@ def is_quantized(node):
 
             # handle activation post process calls
             if node.op == 'call_module':
-                if node.target.split('.')[-1].startswith('activation_post_process_'):
+                if '_activation_post_process_' in node.target.split('.')[-1]:
```

Comment:
maybe put this in a function?
also i'm not sure whether it's a good idea to check the name actually after a second thought, can we check the type instead?